com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJoint.cs

```diff
@@ -45,7 +45,7 @@ public enum JointTypes
         public string jointName;
 
         public abstract JointTypes JointType { get; } // Clear out syntax
-        public bool IsRevoluteOrContinuous => JointType == JointTypes.Revolute || JointType == JointTypes.Revolute;
+        public bool IsRevoluteOrContinuous => JointType == JointTypes.Revolute || JointType == JointTypes.Continuous;
```

Comment:
worth adding a test case?

Reply:
Yeah, good point! I think we eventually need to refactor the urdf joint models and remove the continuous or hinge joints (for RigidBody) from the supported feature.

Reply:
tests added :)